conf/broker.conf

```diff
@@ -140,6 +140,10 @@ authenticationProviders=
 # Enforce authorization
 authorizationEnabled=false
 
+# Actions that can be authorized by using permitted role name which contains wildcard
+# e.g. pulsar.service.*
+wildcardInPermittedRoleNameEnabled=false
```

Comment:
Last comment on this :) 

What about renaming it to something like `authorizationAllowWildcardsMatching` ?